src/components/Dashboard/FilingHistoryList/bodies/FutureEffectivePending.vue

```diff
@@ -24,49 +28,59 @@
 import Vue from 'vue'
 import { Component, Prop } from 'vue-property-decorator'
 import { Getter } from 'vuex-class'
-import { DateMixin } from '@/mixins'
 import { ContactInfo } from '@/components/common'
-import { HistoryItemIF } from '@/interfaces'
+import { ApiFilingIF } from '@/interfaces'
+import { DateUtilities, EnumUtilities } from '@/services'
 
 @Component({
-  components: { ContactInfo },
-  mixins: [DateMixin]
+  components: { ContactInfo }
 })
 export default class FutureEffectivePending extends Vue {
   @Getter getLegalName!: string
 
   /** The subject filing. */
-  @Prop({ required: true }) readonly filing!: HistoryItemIF
+  @Prop({ required: true }) readonly filing!: ApiFilingIF
 
   /** Data for the subject filing. */
   get _ (): any {
-    if (this.filing.isFutureEffectiveIaPending) {
+    if (EnumUtilities.isTypeIncorporationApplication(this.filing)) {
       return {
         subtitle: 'Incorporation Pending',
         filingLabel: 'incorporation'
       }
     }
-    if (this.filing.isFutureEffectiveAlterationPending) {
+    if (EnumUtilities.isTypeAlteration(this.filing)) {
       return {
         subtitle: 'Alteration Pending',
         filingLabel: 'alteration'
       }
     }
+    if (EnumUtilities.isTypeDissolutionVoluntary(this.filing)) {
+      return {
+        subtitle: 'Voluntary Dissolution Pending',
+        filingLabel: 'dissolution'
+      }
+    }
     return {
       subtitle: 'Filing Pending',
       filingLabel: 'filing'
     }
   }
 
-  /** The future effective datetime of the subject filing. */
+  /** The effective date-time of this filing. */
   get effectiveDateTime (): string {
-    return (this.dateToPacificDateTime(this.filing.effectiveDate) || 'Unknown')
+    return this.filing.effectiveDate
+      ? DateUtilities.dateToPacificDateTime(new Date(this.filing.effectiveDate))
```

Comment:
Why do we need to create a new date here? When we create a new date passing to it a parameter of type `FormattedDateTimeGmt`, it will return the correct output needed to convert to pacific date time?

Reply:
`FormattedDateTimeGmt` is actually a string. I created this type to give more meaning to the string.
```
/**
  * Formatted date-time string with explicit GMT timezone (ie, UTC).
  * @example "Tue, 19 Apr 2022 16:40:18 GMT"
  */
export type FormattedDateTimeGmt = string
```

We need to create a new date because it needs to be formatted as a Pacific date.

Reply:
The API provides several time/date formats. In this case we need to convert this:
![image](https://user-images.githubusercontent.com/10002889/228628443-71b3a738-5bb3-44b4-8520-0feeb0d78dbc.png)

to this:
![image](https://user-images.githubusercontent.com/10002889/228628539-52077775-1fd3-42d9-805f-b538159f93d3.png)

Reply:
Oh okay. Thank you so much for the clarification Sev 👍